hack/lib/util.sh

```diff
@@ -308,12 +308,24 @@ kube::util::remove-gen-docs() {
 kube::util::group-version-to-pkg-path() {
   local group_version="$1"
 
-  while IFS=$'\n' read -r api; do
-    if [[ "${api}" = "${group_version/.*k8s.io/}" ]]; then
-      echo "vendor/k8s.io/api/${group_version/.*k8s.io/}"
+  # Make a list of all know APIs by listing their dirs.
+  local apidirs=()
+  kube::util::read-array apidirs < <(
+      cd "${KUBE_ROOT}/staging/src/k8s.io/api"
```

Comment:
shellcheck warns here , it seems you forgot to concatenate the command ?
```
In ./hack/lib/util.sh line 314:
      cd "${KUBE_ROOT}/staging/src/k8s.io/api"
      ^-- SC2164 (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: 
      cd "${KUBE_ROOT}/staging/src/k8s.io/api" || exit
``

Reply:
Fixed